src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -39,6 +39,22 @@ public with sharing class CRLP_RollupUI_SVC {
 
     private static final String SUCCESS_RESPONSE = 'SUCCESS';
 
+    @TestVisible
+    private static final Set<String> fieldsToExclude = new Set<String> {
+        'topinsightid',
```

Comment:
Since you're include multiple versions of activitymetric and activitymetricrollup (with and without `id`), does it make sense to do the same for topinsightid and opportunityscoreid?

Reply:
In the debug logs, the field came up as topinsightid and opportunityscoreid, whereas could not duplicate the other field names, that's why didn't include those. But, I will include just in case.